modules/ROOT/nav.adoc

```diff
@@ -1,6 +1,7 @@
 
 * xref:server-api::page$notifications/v2_schedule.adoc[API Driven Notification Sends]
 * xref:server-api::page$notifications/v2_scheduled_notifications.adoc[Bulk Schedule Creation]
+* xref:server-api::page$notifications/v2_notification_groups.adoc[Notification Groups Management]
```

Comment:
Needs rename to Messages Management, and page name fix.

Reply:
D'oh.